### PR TITLE
Use tide for kubernetes/community

### PR DESCRIPTION
**Pull request by spiffxp**
Strawman PR:
- could we start using tide for k/community as-is?
- what other mungers do we need to migrate over to prow before we could decommission k/community's mungegithub?

### CODE REVIEW

**Comment by spiffxp** [comment]
/hold
oh no I am not letting this accidentally go though

**Comment by cjwagner** [comment]
#5968 will replace the `sig-mention-handler` munger with the `require-sig` plugin for all repos.

/lgtm

**Comment by spiffxp** [comment]
opened #6022 since [we've got a couple `needs-rebase` PR's in kubernetes/community](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+label%3Aneeds-rebase)

File: prow/config.yaml [hunk 1]
```diff
@@ -38,6 +39,7 @@ tide:
     - needs-ok-to-test
     - do-not-merge/work-in-progress
     - do-not-merge/hold
+    - do-not-merge/blocked-paths
```
Comment (cblecker) [inline prow/config.yaml]: should we just add all the `do-not-merge/*` labels in here?

https://github.com/kubernetes/test-infra/blob/661a6b261bcb2004ddede6b935b13629e1a8bf58/label_sync/labels.yaml#L29-L40

**Comment by cblecker** [comment]
We would also still need `owner-label` and `lgtm-after-commit`. Those are both important.

File: prow/config.yaml (hunk 1 above)
Comment (spiffxp) [reply prow/config.yaml]: Fair point.  I'd personally like to get rid of the unprefixed `do-not-merge` label by migrating it to one of the other labels, but that can be done orthogonally.  I'll add it here for now.

**Comment by spiffxp** [comment]
Discussed during sig-testing today, the use of tide won't interfere with continued use of the remaining mungegithub mungers.  We'll open issues to track their migration separately.

Gave kubernetes-dev@ a heads-up https://groups.google.com/forum/#!topic/kubernetes-dev/h-0hGFJ8x1E

**Comment by cblecker** [comment]
@spiffxp The concern I have with this is that the SQ is responsible for protected branches. Do we have a way to do this with tide?

**Comment by cjwagner** [comment]
> Do we have a way to do this with tide?

Not currently.
IIRC we plan to make a cron-job or trusted periodic job that can manage this based on the job config. I'll make an issue.

It isn't completely necessary to automate this so long as a repo admin updates the required contexts when a new job is added to a repo. If the community repo doesn't have plans to add a new job soon then they wouldn't have to do anything at all.

**Comment by cblecker** [comment]
I'm good with this then :)

**Comment by spiffxp** [comment]
/assign @cjwagner 
all yours for the mungegithub bounce

**Comment by spiffxp** [comment]
Retired Submit Queue status context
```
spiffxp@whodis:migratestatus (master %)$ ./migratestatus -retire="Submit Queue" -tokenfile=$HOME/.secrets/.github_token -org=kubernetes -repo=community -logtostderr -modify
I1220 16:27:22.046369   15452 wrappers.go:86] CreateStatus(dry=false) ref:dc04bc083922c8d8aca95171592c3c05f31e97a5: Submit Queue:success
I1220 16:27:22.308150   15452 wrappers.go:86] CreateStatus(dry=false) ref:761e8eab416dc91fe59512063deaf2c8ef0b242a: Submit Queue:success
I1220 16:27:22.496415   15452 wrappers.go:86] CreateStatus(dry=false) ref:20b60c47b6c10814b53d1c59c38475923172c7ac: Submit Queue:success
I1220 16:27:22.726408   15452 wrappers.go:86] CreateStatus(dry=false) ref:c2f7e5b641e1548fe2e491f169c13bbd5cd59ce6: Submit Queue:success
I1220 16:27:23.056177   15452 wrappers.go:86] CreateStatus(dry=false) ref:7805ec02cd17267010bb1a0d52181808a2312920: Submit Queue:success
I1220 16:27:23.282872   15452 wrappers.go:86] CreateStatus(dry=false) ref:3718bdef97026d84bbe08881dbfcacbac3ee278e: Submit Queue:success
I1220 16:27:23.487584   15452 wrappers.go:86] CreateStatus(dry=false) ref:084d40f87a68da7cc32181f2a4fcca76760c0ca9: Submit Queue:success
I1220 16:27:23.726525   15452 wrappers.go:86] CreateStatus(dry=false) ref:c470979dac0e258f2b77460f39c108b00e6ad5b6: Submit Queue:success
I1220 16:27:23.945760   15452 wrappers.go:86] CreateStatus(dry=false) ref:b90f483797e3c8a6e5c6036de682f57889173057: Submit Queue:success
I1220 16:27:24.242617   15452 wrappers.go:86] CreateStatus(dry=false) ref:76938f59175d8ae33e72fd872554e7cb4e1f6d45: Submit Queue:success
I1220 16:27:24.437037   15452 wrappers.go:86] CreateStatus(dry=false) ref:9393e1ac19a6691968835368a0ef7957011250de: Submit Queue:success
I1220 16:27:24.666496   15452 wrappers.go:86] CreateStatus(dry=false) ref:77c4937a647df6b2a60183ea8fe86d8a2841ffa3: Submit Queue:success
I1220 16:27:24.880572   15452 wrappers.go:86] CreateStatus(dry=false) ref:708afbc049836862322c7e852b2532d503b3ffb0: Submit Queue:success
I1220 16:27:25.116180   15452 wrappers.go:86] CreateStatus(dry=false) ref:81b53791b6087424f64ebbe01aa020f9268299f1: Submit Queue:success
I1220 16:27:25.291442   15452 wrappers.go:86] CreateStatus(dry=false) ref:ce0073154396de7170ae6f6526e9df791b1f884b: Submit Queue:success
I1220 16:27:25.508835   15452 wrappers.go:86] CreateStatus(dry=false) ref:56c6cbc35fb5bfd8c24a1f2736ad8a97d960994e: Submit Queue:success
I1220 16:27:25.723846   15452 wrappers.go:86] CreateStatus(dry=false) ref:daffa79f6ea9ea3cfb25692e4912a9a0f2cbcf9e: Submit Queue:success
I1220 16:27:25.976786   15452 wrappers.go:86] CreateStatus(dry=false) ref:41212d7f4c43a03487c6d88aace260763c4297d7: Submit Queue:success
I1220 16:27:26.281953   15452 wrappers.go:86] CreateStatus(dry=false) ref:374067155ccc1923c3cabf8746175e436eb98931: Submit Queue:success
I1220 16:27:26.495354   15452 wrappers.go:86] CreateStatus(dry=false) ref:e5347172a9d12a6227a2c76e360201b4864333f3: Submit Queue:success
I1220 16:27:26.662139   15452 wrappers.go:86] CreateStatus(dry=false) ref:c00e99e57b58ee7c5d73a3e21975c3178a4afaa7: Submit Queue:success
I1220 16:27:26.870693   15452 wrappers.go:86] CreateStatus(dry=false) ref:c3a0682b958ccddc63744e1a436afa32d5d2a47f: Submit Queue:success
I1220 16:27:27.173743   15452 wrappers.go:86] CreateStatus(dry=false) ref:999159f13c6b87ce3847caecb1586381115c0e7b: Submit Queue:success
I1220 16:27:27.407190   15452 wrappers.go:86] CreateStatus(dry=false) ref:62204c38ca8237fed00b623fb6bedc35def3e93b: Submit Queue:success
I1220 16:27:27.619335   15452 wrappers.go:86] CreateStatus(dry=false) ref:58a1ba55effe150b4f5f1e2d770f86ef3041ffd8: Submit Queue:success
I1220 16:27:27.849916   15452 wrappers.go:86] CreateStatus(dry=false) ref:616407bbfabd882d2074625cc60f51d3c25c3c3b: Submit Queue:success
I1220 16:27:28.086814   15452 wrappers.go:86] CreateStatus(dry=false) ref:93a9b85b8e0339c639d5762e87a021c1a0c4799c: Submit Queue:success
I1220 16:27:28.304797   15452 wrappers.go:86] CreateStatus(dry=false) ref:1a6217c00d01f4f24efb36bfc942df9592542e46: Submit Queue:success
I1220 16:27:28.526557   15452 wrappers.go:86] CreateStatus(dry=false) ref:2e96ac32c7afec9556936fe99284ccf7178e225a: Submit Queue:success
I1220 16:27:28.731308   15452 wrappers.go:86] CreateStatus(dry=false) ref:71f3bd2963f7dd4e396674c95b0db4b654e5793c: Submit Queue:success
I1220 16:27:28.911403   15452 wrappers.go:86] CreateStatus(dry=false) ref:cddf6175b13048857a5de0a0d525d67e0eb60bc9: Submit Queue:success
I1220 16:27:29.148719   15452 wrappers.go:86] CreateStatus(dry=false) ref:ef0f99399aa35aa1ef8ed8410df011068b05e298: Submit Queue:success
I1220 16:27:29.495126   15452 wrappers.go:86] CreateStatus(dry=false) ref:1a0f21fc5a18db78225b4a34b41c47292c61054d: Submit Queue:success
I1220 16:27:29.707122   15452 wrappers.go:86] CreateStatus(dry=false) ref:3a8b03413d4a913b1dc1baa32d9c6ee87134c466: Submit Queue:success
I1220 16:27:29.934551   15452 wrappers.go:86] CreateStatus(dry=false) ref:46456058c331dfc819e9e0095b4d20e5520b677a: Submit Queue:success
I1220 16:27:30.145576   15452 wrappers.go:86] CreateStatus(dry=false) ref:911b386903bff7f5aedfafd4bfddd645036e48ab: Submit Queue:success
I1220 16:27:30.362021   15452 wrappers.go:86] CreateStatus(dry=false) ref:5717b9554f6c3e021a5d1f368a94132aed563fd5: Submit Queue:success
I1220 16:27:30.576296   15452 wrappers.go:86] CreateStatus(dry=false) ref:1ab70746a31575573c34c1931e7516c83b591dc1: Submit Queue:success
I1220 16:27:30.819947   15452 wrappers.go:86] CreateStatus(dry=false) ref:7290e161d42c05277a63df97a051f27f3fcc31c0: Submit Queue:success
I1220 16:27:31.059477   15452 wrappers.go:86] CreateStatus(dry=false) ref:96f7946dd49ab99e7c96925394400a7c48b2bdbc: Submit Queue:success
I1220 16:27:31.294486   15452 wrappers.go:86] CreateStatus(dry=false) ref:474cd489c4258214392bdc77661c6d3340d1eac8: Submit Queue:success
I1220 16:27:31.502452   15452 wrappers.go:86] CreateStatus(dry=false) ref:61eafa30f13070f713976205e708c18cadc54434: Submit Queue:success
I1220 16:27:31.706843   15452 wrappers.go:86] CreateStatus(dry=false) ref:88f17ae8e227257d9dfad2546d64553beb8e678b: Submit Queue:success
I1220 16:27:31.936177   15452 wrappers.go:86] CreateStatus(dry=false) ref:324d750f418ea4d88ae9be0fae73ffbf1c1070a3: Submit Queue:success
I1220 16:27:32.238497   15452 wrappers.go:86] CreateStatus(dry=false) ref:14c53fc18238fed19313707847556acc4714e8df: Submit Queue:success
I1220 16:27:32.483511   15452 wrappers.go:86] CreateStatus(dry=false) ref:410c57516b195a1d1c42252479ff6c41dbd8722c: Submit Queue:success
I1220 16:27:32.738985   15452 wrappers.go:86] CreateStatus(dry=false) ref:3b8d50c1b24fb310037b265d39374eb0861e4f68: Submit Queue:success
I1220 16:27:32.970575   15452 wrappers.go:86] CreateStatus(dry=false) ref:b40cc8e94b485501a46be3b20a5dc4b677351794: Submit Queue:success
I1220 16:27:33.189043   15452 wrappers.go:86] CreateStatus(dry=false) ref:f382c70296bba845a2c7999f48d62f7764566ee2: Submit Queue:success
I1220 16:27:33.410573   15452 wrappers.go:86] CreateStatus(dry=false) ref:f3a42fd2cec64be4c88e632af42ddfc62d631f4f: Submit Queue:success
I1220 16:27:33.684082   15452 wrappers.go:86] CreateStatus(dry=false) ref:895e25f6a3c82790734fc3aa39f72df529d60af4: Submit Queue:success
I1220 16:27:34.020269   15452 wrappers.go:86] CreateStatus(dry=false) ref:b40a6921178542dc1341a042db1965c039fea670: Submit Queue:success
I1220 16:27:34.256584   15452 wrappers.go:86] CreateStatus(dry=false) ref:3a574407236428071621b19811476114b09e05cc: Submit Queue:success
I1220 16:27:34.504705   15452 wrappers.go:86] CreateStatus(dry=false) ref:41726548ea485aad84edfafc4b32469c81016411: Submit Queue:success
I1220 16:27:34.873853   15452 wrappers.go:86] CreateStatus(dry=false) ref:349f71eaf3e629a87783ec510e20f9ae314636d8: Submit Queue:success
I1220 16:27:35.084655   15452 wrappers.go:86] CreateStatus(dry=false) ref:53c2a804aa6fc936baa2bf35e854c62737b58dba: Submit Queue:success
I1220 16:27:35.301341   15452 wrappers.go:86] CreateStatus(dry=false) ref:67f5049618f952f45a787da770eefda63c351e09: Submit Queue:success
I1220 16:27:35.528455   15452 wrappers.go:86] CreateStatus(dry=false) ref:a64684ea07e62dfebfea9107ca1f824f46b3ad63: Submit Queue:success
I1220 16:27:35.721704   15452 wrappers.go:86] CreateStatus(dry=false) ref:1eaf775381bbd6d3c6e32816144beba1bca807b4: Submit Queue:success
I1220 16:27:35.971230   15452 wrappers.go:86] CreateStatus(dry=false) ref:3418b4e4c6358f5dc747a37b90a97bc792f159ee: Submit Queue:success
I1220 16:27:36.276333   15452 wrappers.go:86] CreateStatus(dry=false) ref:23cc21a5072afd0d3bdeefe52d159f7a07fee8d6: Submit Queue:success
I1220 16:27:36.497479   15452 wrappers.go:86] CreateStatus(dry=false) ref:6eed714b90f8a094aba3e7fde7a476947056eabc: Submit Queue:success
I1220 16:27:36.690632   15452 wrappers.go:86] CreateStatus(dry=false) ref:885bb7d0ee04ce3c18be665541e54678fd521e51: Submit Queue:success
I1220 16:27:36.903013   15452 wrappers.go:86] CreateStatus(dry=false) ref:1063fd6349f1f89929d74460db89c4ce1f259c68: Submit Queue:success
I1220 16:27:37.128122   15452 wrappers.go:86] CreateStatus(dry=false) ref:bbf21c8c82bec1a4bc658455c8049b2e6b9a1504: Submit Queue:success
I1220 16:27:37.364271   15452 wrappers.go:86] CreateStatus(dry=false) ref:9e81de385df8355af78b32ec0dd596cecda86a8a: Submit Queue:success
I1220 16:27:37.566150   15452 wrappers.go:86] CreateStatus(dry=false) ref:7c9583ec81b8b3bc973c1e8d73c12d68e8967064: Submit Queue:success
I1220 16:27:37.794528   15452 wrappers.go:86] CreateStatus(dry=false) ref:e76ef5217eb4cf8329879b38b7575843157197cd: Submit Queue:success
I1220 16:27:38.033186   15452 wrappers.go:86] CreateStatus(dry=false) ref:566a3f4dd848e03ef9db0669069a1512ef97fadc: Submit Queue:success
I1220 16:27:38.222717   15452 wrappers.go:86] CreateStatus(dry=false) ref:890ad1ee4848596e05251b6ee306926bc8c1a14b: Submit Queue:success
I1220 16:27:38.471839   15452 wrappers.go:86] CreateStatus(dry=false) ref:e73a9b2787da5b555862bf17d9cc5a1af487a814: Submit Queue:success
I1220 16:27:38.693658   15452 wrappers.go:86] CreateStatus(dry=false) ref:b44542583ee22599fd0be31862d516aa17a83a0b: Submit Queue:success
I1220 16:27:38.915437   15452 wrappers.go:86] CreateStatus(dry=false) ref:910fe8d5b9e80cdef2ce2bca4f1aa6ef118ee7ad: Submit Queue:success
I1220 16:27:39.071562   15452 wrappers.go:136] ForEachPR processed page 1/0
```

**Pull request merged**